dts/bindings/sensor/invensense,icm20948.yaml

```diff
@@ -0,0 +1,153 @@
+#  Copyright (c) 2023, Lazaro O'Farrill
+#  email: lazaroofarrill@gmail.com
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  ICM20948 Motion sensor by invensense
+
+compatible: "invensense,icm20948"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  int-gpios:
+    type: phandle-array
+    description: |
+      The INT signal default configuration is active-high. The
+      property value should ensure the flags properly describe the
+      signal that is presented to the driver.
+
+  accel-fchoice:
+    type: int
+    description: |
+      Toggle accelerometer filter in low-power mode.
+      Maps to ACCEL_FCHOICE field in ACCEL_CONFIG.
+    default: 0
```

Comment:
Aint the default value 1?

Reply:
It's not.

Reply:
Reset Value: 0x01 of Register ACCEL_CONFIG. Since bit 0 is ACCEL_FCHOICE, default value should be one, or?

Reply:
Sorry. Forgot to switch the bank first when testing it. You are right it is 1.